### PR TITLE
Finish second task

### PR DESCRIPTION
**Pull request by DubeiA**
Waiting for check

### CODE REVIEW

File: routes/api/contacts.js [hunk 1]
```diff
@@ -1,25 +1,89 @@
...
+  }
+});
 
-module.exports = router
+router.delete("/:contactId", async (req, res, next) => {
+  try {
+    const { contactId } = req.params;
+
+    const result = await contacts.removeContact(contactId);
+    console.log(result);
```
Comment (ceREAL133) [inline routes/api/contacts.js]: i don't think this log is necessary here. Please delete it

File: models/contacts.js [hunk 2]
```diff
@@ -1,19 +1,59 @@
...
+    ...data,
+  };
+  contacts.push(newContact);
+  await fs.writeFile(contactsPath, JSON.stringify(contacts, null, 2));
+  return newContact;
+};
+
+const updateContact = async (id, body) => {
+  const contacts = await listContacts();
+  const idx = contacts.findIndex((item) => item.id === id);
```
Comment (ceREAL133) [inline models/contacts.js]: I don't think idx is proper name for constant, rename it please

**Comment by ceREAL133** [comment]
waiting for comments resolve

File: routes/api/contacts.js (hunk 1 above)
Comment (DubeiA) [reply routes/api/contacts.js]: get off

File: models/contacts.js (hunk 2 above)
Comment (DubeiA) [reply models/contacts.js]: rename on searchContact

**Pull request merged**